demo/src/config/uriTemplates.js

```diff
@@ -28,6 +28,7 @@ registerGlobals({ web_base_url: webBaseUrl });
 window.DAM_BASE_URL = damBaseUrl;
 window.IMAGE_BASE_URL = imageBaseUrl;
 window.VIDEO_BASE_URL = ovpBaseUrl;
+window.WEB_BASE_URL = webBaseUrl;
```

Comment:
@rsumilang what's this for?

Reply:
@rsumilang closing this, there's no use case for that window global.  the others are there for the damUrl.js which uses a different strategy to generate urls

Reply:
@gdbrown 
![CleanShot 2024-08-07 at 17 31 13](https://github.com/user-attachments/assets/8a0e34bc-8608-4a6d-95d4-e2823c457989)

Reply:
@gdbrown  ![CleanShot 2024-08-07 at 17 34 12](https://github.com/user-attachments/assets/f82f8372-e4c6-468f-b386-e9b5cf8b74c1)

Reply:
@rsumilang site specific, your uriTemplates file can certainly do that but that banner is not present here.

Reply:
@gdbrown would you like my proposal in the form of a PR?

Reply:
@rsumilang send a more complete screenshot (over slack if oyu want) and some code example of status is doing, is that just a link?

also, is this different per environment?  how so if so why so.